protractor/helpers/reporter.ts

```diff
@@ -1,10 +1,19 @@
 import { SpecReporter, StacktraceOption } from 'jasmine-spec-reporter';
 
+const { AwesomeReport } = require('jasmine-awesome-report');
+
+const config = {
+  fullPath: 'awesome-report',
+  fileName: 'report',
```

Comment:
change this config, to meet the criteria on point 16.2 (La carpeta debe llamarse reports y el reporte awesome)